C/ConnectFourSolver/build_tarballs.jl

```diff
@@ -2,42 +2,35 @@
 # `julia build_tarballs.jl --help` to see a usage message.
 using BinaryBuilder, Pkg
 
+julia_version = v"1.5.3"
+
 name = "ConnectFourSolver"
 version = v"0.1.0"
```

Comment:
This version must be changed before this PR can be merged.

Reply:
@findmyway I am afraid this PR cannot be merged as-is, because we *must* change the version (otherwise the new dependencies for a package with unchanged version will confuse the package registry).

One option would be to switch to 0.1.1 or 0.2.0, as it looks like upstream doesn't really use any versions? If I am mistaken here, a more complicated scheme could be adopted, as explained [here](https://github.com/JuliaPackaging/Yggdrasil/pull/2327#discussion_r550361029).

Reply:
I think `v0.2.0` is enough for this package.